snippets/dig.md

```diff
@@ -0,0 +1,56 @@
+### dig
+
+Return the value in a nested JSON object by the key.
+
+Given the key name (or target), it will look up the key in the object recursively and return the first value if found.
+
+```
+const dig = (obj, target) => {
+  if (!obj) return;
```

Comment:
Type and error checking is something we avoid doing usually. I'd rather have this line removed.